src/components/v5/shared/Stepper/partials/StepperButton/consts.ts

```diff
@@ -0,0 +1,8 @@
+import { StepStage } from './types';
+
+export const iconNameMap: Record<StepStage, string | undefined> = {
```

Comment:
You can wrap this with `Partial` instead of using `undefined` unless you need all the keys here for some reason. If you do can you explain why?

Reply:
I think name should be camel and upper case